other/bootstrap_daemon/README.md

```diff
@@ -1,27 +1,25 @@
-##Instructions for Debian
+##Instructions 
 
 For security reasons we run the daemon under its own user.
 Create a new user by executing the following:
 ```sh
-sudo useradd --system --shell /sbin/nologin --comment "Account to run Tox's DHT bootstrap daemon" --user-group tox-bootstrapd
+sudo useradd --home-dir /var/lib/tox-bootstrapd --create-home --system --shell /sbin/nologin --comment "Account to run Tox's DHT bootstrap daemon" --user-group tox-bootstrapd
```

Comment:
Is it necessary? I'm think, that it will confusing instructions like
        useradd .....
        mkdir /var/lib/tox-bootstrapd
        chown ....

As I understand - /var/lib may not exists in some kind specialized distros - like embedded systems etc.? I never seen this in generel purpose distributions.